doc/tutorial/Client.lhs

```diff
@@ -161,7 +161,7 @@ The types of the arguments for the functions are the same as for (server-side) r
 ## Changing the monad the client functions live in
 
 Just like `hoistServer` allows us to change the monad in which request handlers
-of a web application live in, we also have `hoistClient` for changing the monad
```

Comment:
There was duplicate `in` (the monad **in** which request handlers ... live **in**)